packages/cactus-test-plugin-htlc-eth-besu-erc20/src/test/typescript/integration/api-surface.ts

```diff
@@ -0,0 +1,9 @@
+const tap = require("tap");
```

Comment:
@jordigiam Please migrate this to tape as well, example syntax is in the other tests such as `packages/cactus-test-plugin-htlc-eth-besu-erc20/src/test/typescript/integration/plugin-htlc-eth-besu-erc20/get-single-status-endpoint.test.ts`

Reply:
Done!